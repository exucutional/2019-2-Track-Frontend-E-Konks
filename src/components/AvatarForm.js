import React from 'react';
import styled from '@emotion/styled';

const Avatar = styled.img`
	display: flex;
	height: 8vh;
    filter: invert(0.5);
    background-color: 
    border-radius: 30px;
    margin-top: 20px;
	&:hover {
		filter: invert(0.8);
    }
`;

function AvatarForm() {
	return <Avatar src="https://image.flaticon.com/icons/svg/918/918761.svg"/>;
}

export default AvatarForm;