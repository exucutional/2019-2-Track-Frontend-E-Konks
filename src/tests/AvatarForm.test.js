import React from 'react';
import renderer from 'react-test-renderer';
import AvatarForm from '../components/AvatarForm'

it('renders correctly', () => {
	const tree = renderer
		.create(<AvatarForm/>)
		.toJSON();
	expect(tree).toMatchSnapshot();
});
