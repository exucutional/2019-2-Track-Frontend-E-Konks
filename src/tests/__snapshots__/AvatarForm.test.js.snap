// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<img
  className="css-nc47br"
  src="https://image.flaticon.com/icons/svg/918/918761.svg"
/>
`;
